process/project-lifecycle.md

```diff
@@ -0,0 +1,159 @@
+          - [Gets - Benefits](#gets---benefits)
+          - [Sandbox Entry Requirements](#sandbox-entry-requirements)
+          - [Project Graduation Process: Sandbox to Incubating](#project-graduation-process-sandbox-to-incubating)
+          - [Submission Forms for Sandbox](#submission-forms-for-sandbox)
+          - [Submission Forms for Sandbox to Incubating](#submission-forms-for-sandbox-to-incubating)
+      - [Incubating](#incubating)
+          - [Gives - Review and Reporting](#gives---review-and-reporting)
+          - [Gets - Benefits](#gets---benefits)
+          - [Incubating Entry Requirements](#incubating-entry-requirements)
+          - [Project Graduation Process: Incubating to Graduation](#project-graduation-process-incubating-to-graduation)
+          - [Submission Forms for Incubating](#submission-forms-for-incubating)
+          - [Submission Forms for Incubating to Graduation](#submission-forms-for-incubating-to-graduation)
+      - [Graduated](#graduated)
+          - [Gives - Review and Reporting](#gives---review-and-reporting)
+          - [Gets - Benefits](#gets---benefits)
+          - [Project Graduation Process: Graduated to Archiving](#project-graduation-process-graduated-to-archiving)
+          - [Submission Forms for Graduated to Archived](#submission-forms-for-graduated-to-archived)
+      - [Archived](#archived)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+Projects have a maturity level of sandbox, incubating, or graduated. Archived is for Projects no longer in active development. The maturity level is a signal by OpenSSF as to what sorts of enterprises should be adopting different projects. Projects increase their maturity by demonstrating their sustainability to OpenSSF’s Technical Advisory Committee(TAC): that they have adoption, a healthy rate of changes, committers from multiple organizations, have adopted the OpenSSF Code of Conduct, and have achieved and maintained the Core Infrastructure Initiative Best Practices Badge.
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if (and only if) this is desired
+* Nurture Projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct and IP Policy requirements.
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Sandbox Entry Requirements
+
+Projects being submitted to the OpenSSF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence.
```

Comment:
I feel this is too strong of a requirement for an early stage project (I do agree that this is required for mature, company owned projects). We should keep as-is.

Reply:
I've left it as-is for now.

Reply:
I think the Sandbox entry requirements and process (eg how due diligence is defined at this stage) is important for putting whatever the OpenSSF's vision for a "project landscape" is into practice. If the OpenSSF intends to be more opinionated, and donation/new projects is done _in service_ of supporting upstream security changes, then I think a very lenient Sandbox entry would be misleading to maintainers (whose projects may be stuck in sandbox indefinitely). If the intention is to cultivate many new technologies in the space (a la CNCF), then a lenient Sandbox entry is more appropriate. In general, I have mostly heard the former being OpenSSF's POV on project donation, but I don't think I have yet seen this formalized through the TAC or GB.

Reply:
Defining the scope/vision of the OpenSSF, so that we might evaluate projects against that, is a good discussion for the TAC to have. However, my initial point that IP Transfer is required for the sandbox stage stands. 

> I feel this is too strong of a requirement for an early stage project

@inferno-chromium , this isn't about feelings -- it is a legal matter. The LF cannot accept a pre-existing project without an IP Transfer agreement in place between the contributing company and the LF. Whomever holds the copyright on the project code, or trademark on its name, regardless of its maturity, must come to an agreement with the LF... or it is not an LF-managed project, and therefore would not be part of the OpenSSF and should not be using the OpenSSF name or brand.

Reply:
@AevaOnline - Be respectful in your replies and please don't try to enforce your opinion over others. Lets other TAC members chime in as well. A definition of early stage project is important here and that will decide what needs to be put in place.